### ISSUE TITLE
Add checkbox to OSARA Configuration dialog: "Report actions related to time movement when transport is rolling" (checked by default)

### ISSUE THREAD

**Issue by ScottChesworth** [jcsteh/osara#344]
This was proposed in the comments of #285. Similar functionality has already been implemented by Jim as part of his JAWS scripts, and has been welcomed by JAWS users on the whole. I think its default state should be checked to start with until I can gauge how widely the change of behaviour is being adopted by users.

@jcsteh, how can I be most useful on this one? Would it be helpful for me to whip up a list of relevant actions with IDs? Is there anything else I can do to help?

**Comment by ScottChesworth** [comment]
OMG yes! Huge improvement for loads of my workflows. Thanks!

I wonder, could the tab order be tweaked slightly in the OSARA Config dialog? If the positions of "Report FX when moving to tracks/takes" and "Report transport state (play, record, etc.)" were switched around, then all the timeline/transport related stuff would be together.

**Comment by jcsteh** [comment]
I put the new option near scrub since it seemed similar. I guess time and
transport are similar too, though it's a slippery slope. I might get to it
at some point, though honestly, fiddling with the dialog parts of OSARA is
tedious as hell - that's one reason I rarely get to adding new settings :)
- so I'm not making any promises as to when.

**Comment by ScottChesworth** [comment]
Sure, understood.
